fulp_modules/jobs/brigdoc/code/brigdoc.dm

```diff
@@ -0,0 +1,59 @@
+/datum/job/fulp/brigdoc
+	title = "Brig Physician"
+	auto_deadmin_role_flags = DEADMIN_POSITION_SECURITY
+	department_head = list("Head of Security", "Chief Medical Officer")
+	faction = "Station"
+	total_positions = 2
+	spawn_positions = 1
+	supervisors = "the head of security and the chief medical officer"
+	selection_color = "#ffeef0"
+	minimal_player_age = 14
+	exp_requirements = 300
+	exp_type = EXP_TYPE_CREW
+	exp_type_department = EXP_TYPE_SECURITY
+	id_icon = 'fulp_modules/jobs/cards.dmi'
+	hud_icon = 'fulp_modules/jobs/huds.dmi'
+	fulp_spawn = /obj/effect/landmark/start/brigdoc
+
+	outfit = /datum/outfit/job/brigdoc
+
+	access = list(ACCESS_SECURITY, ACCESS_SEC_DOORS, ACCESS_BRIG, ACCESS_MEDICAL, ACCESS_MORGUE, ACCESS_SURGERY, ACCESS_PHARMACY, ACCESS_CHEMISTRY, ACCESS_MECH_MEDICAL, ACCESS_MINERAL_STOREROOM, ACCESS_WEAPONS)
+	minimal_access = list(ACCESS_SEC_DOORS, ACCESS_BRIG, ACCESS_MEDICAL, ACCESS_MORGUE, ACCESS_SURGERY, ACCESS_MECH_MEDICAL, ACCESS_MINERAL_STOREROOM, ACCESS_WEAPONS)
```

Comment:
do they have the access required to access the surgery tools in the brig on all maps?

Reply:
They get a full med belt with surgery tools